external-crates/move/crates/move-compiler/tests/move_2024/typing/macro_arg_by_name_invalid_usage_value.exp

```diff
@@ -30,3 +18,11 @@ error[E03006]: unexpected name in this position
   = Macro parameters are always treated as value expressions, and are not modified by path operations.
     Path operations include 'move', 'copy', '&', '&mut', and field references
 
+warning[W09005]: dead or unreachable code
+  ┌─ tests/move_2024/typing/macro_arg_by_name_invalid_usage_value.move:9:14
+  │
+9 │         foo!(0);
```

Comment:
intentional?

Reply:
No, the error thing caused it to shift. Thanks for catching it!